Makefile

```diff
@@ -128,7 +128,7 @@ tidy:
 checktidy:
 	./etc/checktidy.sh
 
-checkgenerate:
+checkgenerate: libflux/scanner.c
```

Comment:
I'm curious why you needed to add this?  Or, maybe adding `scanner.c` to `GENERATED_TARGETS` means you won't need to add it here?